.coveragerc

```diff
@@ -0,0 +1,31 @@
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AccessDenied
+    raise AssertionError
+    raise NotImplementedError
+    raise TunnelError
+
+    # Don't complain about configuration code
+    if os\.getenv\(\"FLASK_ENV\"\) == "production":
+    if current_app\.config\[\"ENV\"\] == "production": 
+    if current_app\.config\[\"ENV\"\] == "development": 
```

Comment:
Is this a duplicate line?

Reply:
There's a line for `development` and one for `production`